exonum-java-binding/core/rust/exonum-java/src/node.rs

```diff
@@ -45,7 +45,8 @@ pub fn run_node(command: Command) -> Result<(), failure::Error> {
 
 fn create_node(config: Config) -> Result<Node, failure::Error> {
     let node_config = config.run_config.node_config.clone();
-    let events_pool_capacity = &node_config.private_config.mempool.events_pool_capacity;
+    // TODO: rewrite with NodeBuilder API when it is available
+    let events_pool_capacity = &Default::default();
```

Comment:
Unfortunately, we can't use events_pool_capacity from user's config in current Core version, but it is addressed in the latest Core revision.